docs/release-notes/release-notes-0.18.5.md

```diff
@@ -50,6 +50,9 @@
 
 * [Improved user experience](https://github.com/lightningnetwork/lnd/pull/9454)
  by returning a custom error code when HTLC carries incorrect custom records.
+
+* [Make input validation stricter](https://github.com/lightningnetwork/lnd/pull/9470)
+  when using the `BumpFee` RPC.
```

Comment:
I think we should also mention the changed behavior - a new param `deadline-delta` is introduced, and if the user wants to change the deadline they now must also specify a budget. The old `target-conf` is now used for estimating the starting fee rate instead of being treated as the deadline, and it cannot be set together with `StartingFeeRate`.